lib/eddystone-beacon-scanner.js

```diff
@@ -1,15 +1,23 @@
 var events = require('events');
 var util = require('util');
 
-var debug = require('debug')('eddystone-beacon-scanner');
+var debug = require('debug')('uri-beacon-scanner');
```

Comment:
Still waiting on approval from @sandeepmistry  on this PR, but yes this needs to be changed.